controllers/pkg/reconcilers/bootstrap-packages/reconciler.go

```diff
@@ -89,144 +82,135 @@ func (r *reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, c a
 type reconciler struct {
 	client.Client
 	porchClient client.Client
-
-	l logr.Logger
+	log         logr.Logger
```

Comment:
we can remove this and use the log.FromContext.

so do this:
log := log.FromContext(ctx)
iso
r.log =  log.FromContext(ctx)

Reply:
r.log is being used in the getPrResources and sub function so how to access it from there? Pass it as a param?

Reply:
if you pass the context you can do the same there. The controller SDK has this as a mechanism. You pass the logger via the ctx and you consume it through the ctx.

Reply:
Done